composer.json

```diff
@@ -50,7 +50,7 @@
         ]
     },
     "suggest": {
-        "ext-curl": "For (web) reports extension",
+        "ext-curl": "For (web) reports extension.",
```

Comment:
this can actually be removed there is no longer a (web) reports extension :eyes:

Reply:
Done